agent/agent-tooling/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/localstorage/LocalFilePurger.java

```diff
@@ -102,7 +96,7 @@ private void purgedExpiredFiles(File folder) {
   private boolean expired(String fileName) {
     String time = fileName.substring(0, fileName.lastIndexOf('-'));
     long milliseconds = Long.parseLong(time);
-    Date twoDaysAgo = new Date(System.currentTimeMillis() - 1000 * expiredInterval);
+    Date twoDaysAgo = new Date(System.currentTimeMillis() - 1000 * expiredIntervalSeconds);
```

Comment:
NIT: I think expirationDate would be a good name.  If expiredInterval is changed for some reason to say 3 days, this would be incorrect.  _